src/openforms/registrations/contrib/objects_api/plugin.py

```diff
@@ -88,10 +88,15 @@ def register_submission(
         """
 
         self.set_defaults(options)
+        log = logger.bind(
+            handler_version=options["version"],
+            public_reference=submission.public_registration_reference,
+        )
 
         handler = HANDLER_MAPPING[options["version"]]
 
         handler.save_registration_data(submission, options)
+        log.debug("registration_data_saved")
```

Comment:
I do not see an exception handler in the `save_registration_data`, are we sure that `registration_data_saved` or should we have something like "trying" to save the data?

Reply:
yeah so if it crashes, the log will never be emitted :)